ush/oceanice_nc2grib2.sh

```diff
@@ -235,7 +235,9 @@ component=${1:?"Need a valid component; options: ice|ocean"}
 grid=${2:-"0p25"} # Default to 0.25-degree grid
 current_cycle=${3:-"2013100100"} # Default to 2013100100
 avg_period=${4:-"0-6"} # Default to 6-hourly average
-ocean_levels=${5:-"5:15:25:35:45:55:65:75:85:95:105:115:125"} # Default to 12-levels
+ocean_levels=${5:-"5:15:25:35:45:55:65:75:85:95:105:115\
```

Comment:
I think we should keep the default to under 25 (under `267m`) so that the tests will pass for `mx500`.
In ops, the levels are provided via the yaml, so we should make sure they are the correct ones.